buildchain/buildchain/versions.py

```diff
@@ -363,6 +363,106 @@ def _version_prefix(version: str, prefix: str = 'v') -> str:
 PACKAGES_MAP = {pkg.name: pkg for pkg in RPM_PACKAGES}
 
 
+DEB_PACKAGES = (
```

Comment:
This commit should probably be removed (`DEB_PACKAGES` is already defined l.466)

Reply:
Oops, indeed. Rebase gone wrong